lib/response.js

```diff
@@ -59,17 +59,7 @@ export default class PodiumClientResponse {
     }
 
     [inspect]() {
-        return {
-            referrerpolicy: this.referrerpolicy,
-            crossorigin: this.crossorigin,
-            integrity: this.integrity,
-            nomodule: this.nomodule,
-            value: this.value,
-            async: this.async,
-            defer: this.defer,
-            type: this.type,
-            data: this.data,
-        };
+        return this.toJSON();
```

Comment:
This was my fix for https://github.com/podium-lib/issues/issues/62

Reply:
Its also fixed in the `next` branch now. Could we update from this with the changes in `next`?

Reply:
Done